thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/ThirdEyeAnomalyConfiguration.java

```diff
@@ -105,14 +122,6 @@ public SmtpConfiguration getSmtpConfiguration() {
     return smtpConfiguration;
   }
 
-  public boolean isMerger() {
-    return merger;
-  }
-
-  public void setMerger(boolean merger) {
-    this.merger = merger;
-  }
```

Comment:
did you rebase to the latest master? this looks like it doesn't belong here

Reply:
Yen let me remove the deprecated isMerger flag since I'm modifying this part of this file.

Reply:
ok